examples/echo.rs

```diff
@@ -1,59 +1,64 @@
 extern crate futures;
-extern crate tokio;
+extern crate tokio_core as tokio;
```

Comment:
I think the code reads easier (from an educational point of view) if you avoid 'as *' in this crate.